src/dpsvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/dfcsm/ValidationController.java

```diff
@@ -39,36 +41,34 @@ public ValidationController(DefaultApi ordsDfcmsApi) {
      * code of 200. THIS IS EXPECTED AND WHAT THE LEGACY SYSTEM DOES. PLEASE DO NOT
      * CHANGE THIS. An error is indicated to the calling system by the -2 int
      * value in the response structure.
-     *
-     * @param driversLicense
+     *  @param driversLicense
      * @param surcode
      */
     @RequestMapping(value = "/getValidOpenDFCMCase",
-            produces = {"application/xml"},
+            produces = MediaType.APPLICATION_XML_VALUE,
             method = RequestMethod.GET)
-    @ApiOperation(value = "Driver Fitness Case Management Validation Service", notes = "", tags = {"DPSValidationService"})
-    @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "Successful operation")})
-    public String getValidOpenDFCMCase(@ApiParam(value = "driversLicense", required = true) @RequestParam(value = "driversLicense", required = true) String driversLicense, @ApiParam(value = "surcode", required = true) @RequestParam(value = "surcode", required = true) String surcode) {
-        /* TODO implement the logic as in the webmethods with the ORDS pieces in  */
-        logger.info("getValidOpenDFCMCase is called");
-
-        String integer_ = "2"; // This will be changed later to accept the int value from the ORDS
-        String caseDesc = "ROUTINE - PROFESSIONAL"; // This will be changed later to accept the caseDesc value from the ORDS
-
-        boolean match_driversLicense = Pattern.matches("[0-9]{7}", driversLicense);
-        boolean match_surcode = Pattern.matches("^[a-zA-Z&-.]{0,3}", surcode);
-
-        if (!match_driversLicense) {
-            logger.fatal("Invalid driversLicense format.");
-            return String.format(DpsValidationServiceConstants.VALIDOPEN_DFCMCASE_ERR_RESPONSE, DpsValidationServiceConstants.VALIDOPEN_DFCMCASE_ERR_RESPONSE_CD);
+    @ApiOperation(value = "Driver Fitness Case Management Validation Service", notes = "", tags = {
+            "DPSValidationService"})
```

Comment:
is this violating a character limit per line constraint in a linter?

Reply:
yup